src/puppetlabs/puppetdb/http/index.clj

```diff
@@ -0,0 +1,18 @@
+(ns puppetlabs.puppetdb.http.index
+  (:require [net.cgrand.moustache :refer [app]]
+            [puppetlabs.puppetdb.http.query :as http-q]
+            [puppetlabs.puppetdb.middleware :refer [wrap-with-paging-options]]
+            [puppetlabs.puppetdb.query.paging :as paging]))
+
+(defn routes
+  [version optional-handlers]
+  (let [param-spec {:optional paging/query-params
+                    :required ["query"]}]
+    (app
```

Comment:
Discussed with the group, it does add some work to move to compojure (or some other lib) but we're going to proceed with it and get the routing lib and refactor in the next sprint or two.